README.md

```diff
@@ -10,7 +10,7 @@
 
 Bring icons to your VS Code.
 
-![demo](/images/screenshot890x486.gif)
+![demo](https://raw.githubusercontent.com/robertohuertasm/vscode-icons/master/images/screenshot.gif)
```

Comment:
it's `https://raw.githubusercontent.com/vscode-icons/vscode-icons/master/images/screenshot.gif`